src/app/modules/shared-components/components/add-content/add-content.component.ts

```diff
@@ -37,10 +37,10 @@ export class AddContentComponent implements OnInit, OnDestroy {
 
   ngOnInit(): void {
     this.newContentForm.valueChanges.subscribe((change: {newTitle: string, existingTitle: string}) => {
-      if (change.newTitle.length < 3 && change.existingTitle.length < 3) this.state = 'closed';
+      if (change.newTitle.length < 3 && change.existingTitle.length < 3) this.expanded = false;
```

Comment:
you should compute the length of the trimmed string, otherwise typing 3 spaces launches a search and I don't think it will be accepted by the backend.

Reply:
Done